blurb/blurb.py

```diff
@@ -567,7 +567,7 @@ def _parse_next_filename(filename):
         metadata = {"date": fields[0], "nonce": fields[-2], "section": section}
 
         for field in fields[1:-2]:
-            for name in ("bpo",):
+            for name in ("issue",):
```

Comment:
It might be better if we use `gh-issue` or something similar here, especially if one day we end up on another bug tracker.
If instead we don't care about this distinction and we want to keep it generic so that we don't need to update it again, then using `issue` is fine.  Also note that if we just use `gh` there might be some confusion between the GitHub issue and the corresponding PR.

Reply:
Updated to use `gh-issue`.